src/components/atoms/DatasetInfoBox.tsx

```diff
@@ -32,6 +38,23 @@ const DatasetInfoBox: React.FC<DatasetInfoBoxProps> = ({ title, description, tag
                     </Badge>
                 ))}
             </Pane>
+            <Pane marginTop="1rem" display="flex" flexDirection="column">
+                <Text size={300}>
+                    URL: <Link size={300}>{url}</Link>
+                </Text>
+                <Text size={300} marginTop="0.15rem">
+                    Last updated:{' '}
+                    <Strong size={300}>
+                        {`${updatedDate.getDate()}.${updatedDate.getMonth() + 1}.${updatedDate.getFullYear()}`}
```

Comment:
Date-fns has support for a variety of formatting methods, including ["distance to now in words"](https://date-fns.org/v2.21.1/docs/formatDistanceToNow) which might be a bit more human friendly. It is a common pattern to have the date represented as "4 years ago", and when you hover it you get a tooltip of the exact date.

Furthermore, by using date-fns you could format based on the users locale, which improves user experience.

Reply:
As we've already included it as a dep I think we should use it's pwoers.

Reply:
That's a better solution, I will change it 🤠7